src/install.rst

```diff
@@ -6,197 +6,314 @@ Installing Nextstrain
-         conda update nextstrain-cli
+      .. note::
 
-      Download the latest image with the Nextstrain CLI.
+         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for augur <augur:installation/installation>` and :doc:`install auspice via npm <auspice:introduction/install>`.
 
-      .. code-block:: bash
+      1. `Install WSL 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
+      2. Open a WSL terminal by running **wsl** from the Start menu.
+      3. Install Miniconda:
 
-         nextstrain update
+         .. code-block:: bash
 
-Troubleshoot a broken installation
-==================================
+            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh
+            # follow through installation prompts
+            rm Miniconda3-latest-Linux-x86_64.sh
 
-If Conda fails to install or update Nextstrain using the commands above, it's possible that Conda itself is out-of-date or that Conda cannot figure out how to resolve the environment's dependencies.
-Try the following approaches, to fix these broken installations.
+      4. Install mamba on the ``base`` conda environment:
 
-Remove your environment and start from scratch
-----------------------------------------------
+         .. code-block:: bash
 
-Starting from scratch often fixes problems with Conda environments.
-To start over with a new Nextstrain environment, delete your current environment.
+            conda install -n base -c conda-forge mamba --yes
+            conda activate base
 
-.. code-block:: bash
+      5. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-    conda activate base
-    conda env remove -n nextstrain
+         .. code-block:: bash
 
-Then, repeat the installation instructions above, starting with the update of Conda itself.
+            mamba create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-Use Mamba as an alternative to Conda's environment solver
----------------------------------------------------------
+      6. Install the remaining Nextstrain components. There are two options:
 
-`Mamba <https://github.com/mamba-org/mamba>`_ is a drop-in replacement for most ``conda`` functionality that implements a faster dependency solving algorithm in C++ and multithreaded downloads.
-As a result, Mamba can install Conda packages much faster and more accurately than the original Conda installer.
+         a. Docker (recommended) – on Windows, `install Docker Desktop for WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
+
+            - Make sure to follow through the last step of enabling **WSL Integration**.
+
+         b. Native – install all the necessary software using mamba:
+
+            .. code-block:: bash
+
+               mamba install -c conda-forge -c bioconda augur auspice snakemake --yes
```

Comment:
See comment above about including nextalign and git here.

Reply:
Fixed here: https://github.com/nextstrain/docs.nextstrain.org/pull/81/commits/4584bd292626e4c6671d37e5870157369a7557dd